smoketest/config-tests/dialup-router-medium-vpn

```diff
@@ -141,51 +141,51 @@ set interfaces wireguard wg1 peer sam preshared-key 'XpFtzx2Z+nR8pBv9/sSf7I94OkZ
 set interfaces wireguard wg1 peer sam public-key 'v5zfKGvH6W/lfDXJ0en96lvKo1gfFxMUWxe02+Fj5BU='
 set interfaces wireguard wg1 port '7778'
 set nat destination rule 50 destination port '49371'
-set nat destination rule 50 inbound-interface 'pppoe0'
```

Comment:
These configs represent out in the wild VyOS co digs that are known to work.

They should not be changed at all as they serve as input for the migration script. This usually means sonething is odd with the migrator

Reply:
This fixes the config test extension I wrote that checks the resulting config against expected values, not changing the config itself.

Reply:
Ah, dis not know we have an „after migration“ test, very cool